frame/democracy/src/lib.rs

```diff
@@ -528,20 +502,11 @@ pub mod pallet {
 		ValueQuery,
 	>;
 
-	/// Accounts for which there are locks in action which may be removed at some point in the
-	/// future. The value is the block number at which the lock expires and may be removed.
-	///
-	/// TWOX-NOTE: OK ― `AccountId` is a secure hash.
-	#[pallet::storage]
-	#[pallet::getter(fn locks)]
-	pub type Locks<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, T::BlockNumber>;
-
```

Comment:
Why did we remove the Locks? Howcome those are not necessary when going multicurrency?

Reply:
That should be from the merge, the latest substrate democracy pallet does not use it

Reply:
Why not start out this PR from the latest code then, which means we have far fewer diffs?

Reply:
you mean to get the latest substrate democracy pallet ?